.github/workflows/tests.yaml

```diff
@@ -17,6 +17,11 @@ jobs:
           submodules: true
       - name: Install packages
         run: |
+          sudo snap install --no-wait ruff
```

Comment:
Are you leaving this on 5 lines on purpose?
I thought you could do:
```
sudo snap install --no-wait ruff shellcheck woke
sudo snap install --no-wait --classic node pyright
```

Reply:
It works for the non-classic ones, but the classic ones need their own lines, so I opted for consistency.